service-api/app/test/AppTest/DataAccess/ApiGateway/DataStoreLpasTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppTest\DataAccess\Repository;
```

Comment:
```suggestion
namespace AppTest\DataAccess\ApiGateway;
```

Should match the folder the files in.